spec/rubocop/cop/rspec/factory_bot/attribute_defined_statically_spec.rb

```diff
@@ -161,56 +238,4 @@
       end
     RUBY
   end
-
-  bad = <<-RUBY
-    FactoryBot.define do
-      factory :post do
-        title "Something"
-        comments_count 0
-        tag Tag::MAGIC
-        recent_statuses []
-        status([:draft, :published].sample)
-        published_at 1.day.from_now
-        created_at(1.day.ago)
-        updated_at Time.current
-        update_times [Time.current]
-        meta_tags(foo: Time.current)
-        other_tags({ foo: Time.current })
-        options color: :blue
-        other_options Tag::MAGIC => :magic
-        self.end Date.tomorrow
-
-        trait :old do
-          published_at 1.week.ago
-        end
-      end
-    end
-  RUBY
-
-  corrected = <<-RUBY
-    FactoryBot.define do
-      factory :post do
-        title { "Something" }
-        comments_count { 0 }
-        tag { Tag::MAGIC }
-        recent_statuses { [] }
-        status { [:draft, :published].sample }
-        published_at { 1.day.from_now }
-        created_at { 1.day.ago }
-        updated_at { Time.current }
-        update_times { [Time.current] }
-        meta_tags { { foo: Time.current } }
-        other_tags { { foo: Time.current } }
-        options { { color: :blue } }
-        other_options { { Tag::MAGIC => :magic } }
-        self.end { Date.tomorrow }
-
-        trait :old do
-          published_at { 1.week.ago }
-        end
-      end
-    end
-  RUBY
-
-  include_examples 'autocorrect', bad, corrected
```

Comment:
`autocorrect` shared example does not pass the correct file path to `expect_offense`, and thus the cop considers the file irrelevant and skips it, failing the example.